### PR TITLE
Social solidus setup

### PR DESCRIPTION
**Pull request by dcuadraq**
## Setup for solidus_social
Fixes issue with solidus_social not working correctly on dummy app.

### CODE REVIEW

**Comment by dcuadraq** [comment]
I think Im done with this PR @mumoc

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Introduction goes here.
 Installation
 ------------
 
-Make sure to have the extension [solidus_social](https://github.com/solidusio-contrib/solidus_social) installed before adding this one.
+Make sure to have the extension [solidus_social](https://github.com/solidusio-contrib/solidus_social) installed (migrations ran) before adding this one.
```
Comment (mumoc) [inline README.md:9]: /??? `to have solidus social installed and migrations ran before ...` ???

File: spec/controllers/spree/api/social_auths_controller_spec.rb [hunk 2]
```diff
@@ -1,7 +1,23 @@
 require 'spec_helper'
 
 describe Spree::Api::SocialAuthsController do
-  it 'holi' do
-    expect(post: '/api/facebook/signin').to route_to(controller: 'socials', action: 'signin')
+  it 'offers endpoint for amazon' do
+    expect(post: '/api/amazon/signin').to route_to(format: 'json', controller: 'spree/api/social_auths', action: 'amazon_signin')
```
Comment (mumoc) [inline spec/controllers/spree/api/social_auths_controller_spec.rb]: can this be done using the helpers? like

`post: amazon_signin_path` or something? if not, it's ok

**Review by mumoc — APPROVED** [review]
No need to update the comments if you think it is correct as it is right now ^^

File: README.md (hunk 1 above)
Comment (dcuadraq) [reply README.md:9]: In here what I intend to say is that the migrations are the important part of having installed `solidus_social`, do you consider it confusing?

File: spec/controllers/spree/api/social_auths_controller_spec.rb (hunk 2 above)
Comment (dcuadraq) [reply spec/controllers/spree/api/social_auths_controller_spec.rb]: Thanks, didn't think it would work.

**Pull request merged**